src/components/statusHelper.js

```diff
@@ -343,6 +345,87 @@ export const styledConnectionStatus = (status, err) =>
         </Text>
       </TextContent>
     ),
+    no_rhc: (
+      <TextContent>
+        <Text component={TextVariants.p}>
+          Cannot remediate - Cloud connector not defined
+          <Text component={TextVariants.small} style={{ margin: '0px' }}>
+            Remediation from Insights requires Cloud connector. Cloud connector
+            can be enabled via Satelite, or through
+            <Button
+              style={{ padding: '0px' }}
+              key="configure"
+              variant="link"
+              // eslint-disable-next-line max-len
+              href="LINK HERE"
+            >
+              RHC (Red Hat connector)
+            </Button>
+          </Text>
+          <Button
+            style={{ padding: '0px' }}
+            key="download"
+            variant="link"
+            component="a"
+            // eslint-disable-next-line max-len
+            href="LINK HERE"
```

Comment:
NOTE:  SEE BELOW (won't be available until 18th of May)  Docs are still in review.